examples/train_policy.py

```diff
@@ -47,7 +47,7 @@
     parser.add_argument(
         '--total-steps',
         type=int,
-        default=1024000,
+        default=10000000,
```

Comment:
`steps_per_epoch` is default to 2048. Should we update it meanwhile?

Reply:
I have updated all on-policy configs in this pull request, please check them out.